README.md

```diff
@@ -1,149 +1,43 @@
+The example project is part of the [Arm Mbed OS quick start guide](https://os.mbed.com/docs/mbed-os/v5.14/quick-start/index.html). It contains an application to repeatedly blink an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
 
-Please install [Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+The project can be built with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli). Please install Arm Mbed CLI.
 
-## Import the example application
+Depending on the target, the example project can be built with `GCC_ARM`, `ARM` or `IAR` toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain supports your target.
 
-From the command-line, import the example:
-
-```
-mbed import mbed-os-example-blinky
-cd mbed-os-example-blinky
-```
-
-### Now compile
-
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the Arm Compiler:
-
-```
-mbed compile -m K64F -t ARM
+```bash
+$ mbed compile -S
 ```
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
-
-```
-[snip]
-
-Image: ./BUILD/K64F/GCC_ARM/mbed-os-example-blinky.bin
-```
-
-### Program your board
-
-1. Connect your Mbed device to the computer over USB.
-1. Copy the binary file to the Mbed device.
-1. Press the reset button to start the program.
+Clone this repository on your system and change the current directory to where the project was cloned.
 
-The LED on your platform turns on and off. The main thread will additionally take a snapshot of the device's runtime statistics and display it over serial to your PC. The snapshot includes:
 
-* System Information:
-    * Mbed OS Version: Will currently default to 999999
-    * Compiler ID
-        * ARM = 1
-        * GCC_ARM = 2
-        * IAR = 3
-    * [CPUID Register Information](#cpuid-register-information)
-    * [Compiler Version](#compiler-version)
-* CPU Statistics
-    * Percentage of runtime that the device has spent awake versus in sleep
-* Heap Statistics
-    * Current heap size
-    * Max heap size which refers to the largest the heap has grown to
-* Thread Statistics
-    * Provides information on all running threads in the OS including
-        * Thread ID
-        * Thread Name
-        * Thread State
-        * Thread Priority
-        * Thread Stack Size
-        * Thread Stack Space
+## Application functionality
```

Comment:
I would put this section either at the top after the first paragraph that introduces the example or in a section that  describes the expected output.